### DISCUSSION TITLE
Reply if you wanna contribute

### DISCUSSION THREAD

**Discussion by PeerRich** [NostrGit/NostrGit#77] [General]
Jack Dorsey made a bounty

https://news.ycombinator.com/item?id=35020964

We'd love to split it with everyone contributing.

Comment below and go through the [Issues](https://github.com/NostrGit/NostrGit/issues) to see where you can help

**Comment by brycedev** [comment]
i'm in. i've begun some work

**Comment by BruceWangyq** [comment]
Count me in!

**Comment by jamesindeed** [comment]
I'm in 😎

**Comment by cypherhoodlum** [comment]
I'm in. I think all communication should happen openly, using nostr and github. No DMs, No private calls or chats. All right with everyone?

**Comment by PeerRich** [comment]
@cypherhoodlum  i wish nostr supported groups yet. I am all for open! no DMs!

**Comment by PeerRich** [comment]
GitHub until NostrGit is ready 💜

**Comment by PeerRich** [comment]
imagine the day closing this Repo and moving it to nostrgit.com/nostrgit/nostrgit 🤙

**Comment by cypherhoodlum** [comment]
That will be the day.

**Comment by Keyrxng** [comment]
Get me involved with this for sure I'm going to get stuck in and see where I can be of value!

**Comment by PeerRich** [comment]
lets get jack in here!

https://damus.io/note1kgvtaljcqr74dpe599rs6enh4th8vpwvh8pyufpu7z0c3cq35y7s7wcu46

![image](https://user-images.githubusercontent.com/8019099/222992674-dde2b367-ba56-44c7-ab09-a8318cc5ab69.png)

**Comment by bordalix** [comment]
Count me in.

**Comment by mfts** [comment]
Heyo 👋 gonna start contributing

**Comment by cypherhoodlum** [comment]
Add your pubkeys [here](https://github.com/NostrGit/NostrGit/issues/19) if you want to connect on nostr. :rocket: 
@mfts 
@bordalix 
@Keyrxng 
@jamesindeed 
@BruceWangyq 
@brycedev

**Comment by bordalix** [comment]
bordalix@nostr.pt (npub1vt803quxxq32fuwkp42g2lyaw2t9qupvnl3z0vyc3s9kudkyhn8qt28cxv)

**Comment by melvincarvalho** [comment]
I've read through quite a bit of this.  Excellent so far, and I'm supportive of this project

A little backgrounder: I've been working for a couple of years on a system to let machines and humans work together

But back then there was no network that was machine friendly.  Which is why I helped (well, at least a little bit) to build out nostr.  I think the network is now ready to welcome both humans and machines to interact in a cooperative way.

The first aim was to create code bases on github and have them become part of the nostr network, this has evolved into a small first step, which I call NIP-17, tracking commits with nostr

https://github.com/nip17/nips/blob/master/17.md

I like nostrgit.  It's taking on alot, but hopefully I can help a bit.  I'm aiming to take on a small chunk of it, namely commits over nostr, and versioning, distributed consensus which in itself will be hard.  Not sure what will stick, but hopefully ideas can converge!

**Comment by PeerRich** [comment]
great to have you here @melvincarvalho !

**Comment by garyray-k** [comment]
https://github.com/nostr-protocol/nips/pull/223

I'm in. Been trying to rally people around this so glad to find this project. 

https://t.me/nostr_git_context

npub160t5zfxalddaccdc7xx30sentwa5lrr3rq4rtm38x99ynf8t0vwsvzyjc9

**Comment by cypherhoodlum** [comment]
Welcome to the project @melvincarvalho @garyray-k  :heart:

**Comment by ofer-elrom** [comment]
I am in. 
Not sure how much bandwidth I have, but I will try.

**Comment by ofer-elrom** [comment]
> Not sure how much bandwidth I have, but I will try.

I happily give up any share in a possible bounty to all others here. You would be able to contribute much more time than me.

**Comment by ameya-deshmukh** [comment]
I'm in! Browsing through the issues to see where I can help :)

**Comment by nflatrea** [comment]
Let's add another brick in the wall 🧱

**Comment by cypherhoodlum** [comment]
Welcome to the project :)

**Comment by e-roy** [comment]
Hi, I'm interested :) would like to work on #14

**Comment by akhavr** [comment]
https://github.com/akhavr/nostrya though your UI is better :)

**Reply by cypherhoodlum** [reply]
@PeerRich Good to have some competition, eh?

**Reply by cypherhoodlum** [reply]
Not very many around, which is surprising.

**Reply by akhavr** [reply]
I'd better cooperate, since my frontend skills are almost nonexistant :)

**Reply by cypherhoodlum** [reply]
Contributions are welcome :)

**Comment by skeptrunedev** [comment]
Hey, I and @fastfists at @arguflow would be interested in contributing. But why not fork gitea? Wouldn't that be much simpler?

**Reply by cypherhoodlum** [reply]
Hi @kmkhami, forking gitea was tried in the beginning and it turned out to be not such a good idea. It's much simpler to build it piece by piece. We are going fast!

**Reply by cypherhoodlum** [reply]
Your contributions are welcome! :heart:

**Comment by melvincarvalho** [comment]
@kmkhami a possible path, there was some discussion on the federation thread: https://github.com/go-gitea/gitea/issues/18240#issuecomment-1454952558.  Nobody really knows how to add nostr to that codebase.

NostrGit has made a great start, and has momentum

**Reply by Callnote** [reply]
will at least look them over.

**Comment by PeerRich** [comment]
All Discussion moved to Issues: https://github.com/NostrGit/NostrGit/issues/145
